### DISCUSSION TITLE
Permissions for forms/pages

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#13161] [Help]
### Package

Form builder

### Package Version

v3.2.86

### How can we help you?

Is there a built-in way to access control forms? i.e. prevent/allow read/updates

**Comment by leandrocfe** [comment]
What is the goal? Could you provide more info?

**Reply by bilogic** [reply]
Sure, I have a settings page that I would like only authorized users to update. Everyone else can view it. 

It was built by placing a filament form on a filament page.

I found this package https://github.com/bezhanSalleh/filament-shield, but it seems to only prevent the page from loading, I can't find any mechanism that prevents a user from "faking" a POST to the server

**Reply by leandrocfe** [reply]
You can use this in your settings page
https://filamentphp.com/docs/3.x/panels/pages#authorization
https://filamentphp.com/plugins/bezhansalleh-shield#pages

**Reply by bilogic** [reply]
Filament's `canAccess` makes no mention of updating a form, I suppose it does not prevent it, correct?
And for the 2nd link, I don't see any code that prevents updates either.

I think in layman terms, yes, I can't see the form, but what if I cleverly craft a valid POST, will the form update?
